### DISCUSSION TITLE
YCM 0.9.1 Released

### DISCUSSION THREAD

**Discussion by drdanz** [robotology/community#328] [Releases]
YCM 0.9.1 was released today and is now available for download at: https://github.com/robotology/ycm/releases/.

Changes made since YCM 0.9.0 include the following.

### Generic Modules

* `FindOrBuildPackage`: Fixed a typo that made `find_or_build_package` dismiss the first argument passed after the requested package name.

### Find Modules

* The `FindGLFW3` module is now using the `glfw` target exported from upstream if available.
* The `FindESDCANAPI` module is now also compatible with 64 bit libraries.

CMake Next
----------

* Updated to CMake v3.13.0 + commit Kitware/CMake@3804122ab2793b9d54aa8c69df62ed86f526d50f.

3rd Party
---------

* Fixed `gstconfig.h` detection on windows in `FindGStreamer` module.

**Comment by PeterBowman** [comment]
That's v0.9.1 :).

**Comment by drdanz** [comment]
Sorry! Too much copy and paste is required for making a release! :sweat_smile:
